docs/en/user_guides/2_dataset_prepare.md

```diff
@@ -264,6 +264,18 @@ python tools/dataset_converters/hrf.py /path/to/healthy.zip /path/to/healthy_man
 
 The script will make directory structure automatically.
 
+### REFUGE
+
+First, registrate in [REFUGE2020Challenge](https://refuge.grand-challenge.org)and download data from [this link](https://refuge.grand-challenge.org/REFUGE2Download)
+
+To convert REFUGE dataset to MMSegmentation format, you should run the following command:
+
+```shell
+python tools/convert_datasets/refuge.py --raw_data_root=.. --tmp_dir=..
```

Comment:
Make `REFUGE.zip` which could be downloaded from official website as `raw_data` thus users could generate mmsegmentation-style data automatically.